src/metabase/driver/sql_jdbc/sync/describe_table.clj

```diff
@@ -575,51 +576,70 @@
                                         (false? (:json_unfolding existing-field))))]
         (remove should-not-unfold? json-fields)))))
 
+(def ^:dynamic *nested-field-columns-max-row-length*
```

Comment:
wdyt if we make this configurable using env? Would be nice to have an escape hatch in case this is too low / or still too high.

Reply:
Nice idea, I'll take it. We can't really increase this number substantially without risking additional OOMs. So allowing people to increase it if they need to is a sensible choice

Reply:
done in 2d0c2360362d7df400153108fb4c7235d32d041b